dump/config.json

```diff
@@ -0,0 +1,870 @@
+{
+    "P434": {
+        "owl:sameAs": "http://musicbrainz.org/artist/$1"
+    },
+    "P435": {
+        "owl:sameAs": "http://musicbrainz.org/work/$1"
+    },
+    "P436": {
+        "owl:sameAs": "http://musicbrainz.org/release-group/$1"
+    },
+    "P982": {
+        "owl:sameAs": "http://musicbrainz.org/area/$1"
+    },
+    "P966": {
+        "owl:sameAs": "http://musicbrainz.org/label/$1"
+    },
+    "P1004": {
+        "owl:sameAs": "http://musicbrainz.org/place/$1"
+    },
+    "P661": {
+        "owl:sameAs": "http://rdf.chemspider.com/$1"
+    },
+    "P227": {
+        "owl:sameAs": "http://d-nb.info/gnd/$1/about/rdf"
+    },
+    "P244": {
+        "owl:sameAs": "http://id.loc.gov/authorities/names/$1"
+    },
+    "P269": {
+        "owl:sameAs": "http://www.idref.fr/$1/id"
+    },
+    "P349": {
+        "owl:sameAs": "http://id.ndl.go.jp/auth/ndla/$1"
+    },
+    "P646": {
+        "owl:sameAs": "http://freebase.com/$1"
+    },
+    "P662": {
+        "owl:sameAs": "http://pubchem.ncbi.nlm.nih.gov/rest/rdf/compound/$1"
+    },
+    "P214": {
+        "owl:sameAs": "http://viaf.org/viaf/$1"
+    },
+    "P1868": {
```

Comment:
There is no P1868 property on Wikidata? https://www.wikidata.org/wiki/Property:P1868. 
I need also update $1 to $2.